ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesPage.tsx

```diff
@@ -20,25 +21,38 @@ import './WorkloadCvesPage.css';
 
 function WorkloadCvesPage() {
     return (
-        <Switch>
-            <Route path={vulnerabilitiesWorkloadCveSinglePath} component={ImageCvePage} />
-            <Route path={vulnerabilitiesWorkloadCveImageSinglePath} component={ImagePage} />
-            <Route
-                path={vulnerabilitiesWorkloadCveDeploymentSinglePath}
-                component={DeploymentPage}
+        <>
+            <Alert
+                variant="warning"
+                isInline
+                title={
+                    <span>
+                        This is a Technology Preview of this feature. For all production
+                        requirements we recommend using{' '}
+                        <Link to={vulnManagementPath}>Vulnerability Management (1.0)</Link>
```

Comment:
What are pro and con to use Button/LinkShim idiom as elsewhere?

```tsx
                    <Button variant="link" isInline component={LinkShim} href={vulnManagementPath}>
                        Vulnerability Management (1.0)
                    </Button>
```

Reply:
This is something I wanted to bring up during a team meeting, and something that came up during the bug bash. Using the Button idiom makes the text un-selectable, which is the main con from what I can tell. In this case it isn't as much of an issue but it is slightly annoying in the workload cve tables when it applies to CVE Name, Deployment name, etc.

The main pro of the LinkShim idiom is that it avoid Cypress flakes when using a Button-style link from what I remember. Are there other advantages when not using the Button at all?

A larger question - what is the advantage of using a `<Button variant='link'>` instead of just a `<Link>` in most cases?

Reply:
This is something I wanted to bring up during a team meeting, and something that came up during the bug bash. Using the Button idiom makes the text un-selectable, which is the main con from what I can tell. In this case it isn't as much of an issue but it is slightly annoying in the workload cve tables when it applies to CVE Name, Deployment name, etc.

The main pro of the LinkShim idiom is that it avoid Cypress flakes when using a Button-style link from what I remember. Are there other advantages when not using the Button at all?

A larger question - what is the advantage of using a `<Button variant='link'>` instead of just a `<Link>` in most cases?

Reply:
Oh, I did not know that the text becomes un-selectable!

As long as we verify that plain `Link` and corresponding `a` element get correct PatternFly style without an leakage from classic style, then I am 100% in favor of it.

Reply:
Let’s move forward with what you have done here and ask at PatternFly Developer Office hours to understand whether the Button/Link recommendation is obsolete:
https://www.patternfly.org/v4/components/button#router-link